tableauserverclient/server/endpoint/resource_tagger.py

```diff
@@ -0,0 +1,33 @@
+from .endpoint import Endpoint
+from .. import RequestFactory
+from ...models.tag_item import TagItem
+import logging
+import copy
+
+logger = logging.getLogger('tableau.endpoint')
+
+
+class ResourceTagger(Endpoint):
+    # Add new tags to resource
+    def _add_tags(self, baseurl, resource_id, tag_set):
```

Comment:
Will the base url change between invocations to the same instance of a class?  If not (which I don't think it does based on reading the code), pass in baseurl into the constructor of the class.